scripted/src/locales/el/locale.js

```diff
@@ -0,0 +1,253 @@
+    "%database.idProperty.reversePluralLabel": "ids των",
+    "%database.typeProperty.label": "είδος",
+    "%database.typeProperty.pluralLabel": "είδη",
+    "%database.typeProperty.reverseLabel": "είδος του",
+    "%database.typeProperty.reversePluralLabel": "είδη των",
+    "%database.uriProperty.label": "URI",
+    "%database.uriProperty.pluralLabel": "URIs",
+    "%database.uriProperty.reverseLabel": "URI του",
+    "%database.uriProperty.reversePuralLabel": "URIS των",
+    "%database.sortLabels.text.ascending":  "α - ω",
+    "%database.sortLabels.text.descending": "ω - α",
+    "%database.sortLabels.number.ascending":  "το μικρότερο πρώτα",
+    "%database.sortLabels.number.descending": "το μεγαλύτερο πρώτα",
+    "%database.sortLabels.date.ascending":  "το αρχικό πρώτα",
+    "%database.sortLabels.date.descending": "το τελευταίο πρώτα",
+    "%database.sortLabels.boolean.ascending":  "το λάθος πρώτα",
+    "%database.sortLabels.boolean.descending": "το σωστό πρώτα",
+    "%database.sortLabels.item.ascending":  "α - ω",
+    "%database.sortLabels.item.descending": "ω - α",
+    "%database.reverseLabel": "αντιστροφή του αρχείου %1$s",
+    "%database.reversePluralLabel": "αντιστροφή του αρχείου %1$s",
+    "%database.error.unloadable": "Could not load data.",
+    "%database.error.loadTypesFailure": "Database loading of types failed",
+    "%database.error.loadPropertiesFailure": "Database loading of properties failed",
+    "%database.error.loadItemsFailure": "Database loading of items failed",
+    "%database.error.removeAllStatementsFailure": "Removing all statements from database failed",
+    "%database.error.noImporterFailure": "No importer for data of type %1$s",
+    "%database.error.itemSyntaxError": "Item entry has no label and no id: %1$s",
+    "%database.error.itemMissingLabelFailure": "Cannot add new item containing no label: %1$s",
+
+    "%expression.error.noSuchFunction": "No such function named %1$s",
+    "%expression.error.noSuchVariable": "No such variable called %1$s",
+    "%expression.error.mustBeForward": "Last path of segment must be forward",
+    "%expression.error.missingPropertyID": "Missing property ID at position %1$d",
+    "%expression.error.missingFactor": "Missing factor at end of expression",
+    "%expression.error.missingParenEnd": "Missing ) to end %1$s at %2$d",
+    "%expression.error.missingParenStart": "Missing ( to start %1$s at %2$d",
+    "%expression.error.missingParenFunction": "Missing ) after function call %1$s at position %2$d",
+    "%expression.error.missingParen": "Missing ) at position %1$d",
+    "%expression.error.unexpectedSyntax": "Unexpected text %1$s at position %2$d",
+    "%expression.error.unterminatedString": "Unterminated string starting at %1$d",
+
+    "%coders.mixedCaseLabel": "μεικτά",
+    "%coders.missingCaseLabel": "λείπουν",
+    "%coders.othersCaseLabel": "άλλα",
+    "%coders.error.configuration": "%1$s: Σφάλμα επεξεργασίας των ρυθμίσεων του κωδικοποιητή",
```

Comment:
@karger the original says "Error processing configuration of coder", is coder here some sort of encoder or the person writing code? The translation assumes the former, but I suspect it's the latter.